src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -962,6 +992,18 @@ internal sealed override void PostDecodeWellKnownAttributes(ImmutableArray<CShar
                         ErrorCode.WRN_ExternMethodNoImplementation;
                     diagnostics.Add(errorCode, this.Locations[0], this);
                 }
+
+                if (_lazyCustomAttributesBag.EarlyDecodedWellKnownAttributeData is CommonMethodEarlyWellKnownAttributeData { UnmanagedCallersOnlyAttributeData: not null })
+                {
+                    for (int i = 0; i < boundAttributes.Length; i++)
+                    {
+                        if (boundAttributes[i].IsTargetAttribute(this, AttributeDescription.UnmanagedCallersOnlyAttribute))
+                        {
+                            UnmanagedCallersOnlyAttributeErrors(boundAttributes[i], allAttributeSyntaxNodes[i], diagnostics);
+                            break;
```

Comment:
>break [](start = 28, length = 5)

Should we continue and report errors for all `[UnmanagedCallersOnly]` attributes on the method?

Reply:
Probably, I'll add a test.

---
In reply to: [483176341](https://github.com/dotnet/roslyn/pull/47311#discussion_r483176341) [](ancestors = 483176341)